Babylon/groups/config/groups/platform/commands/select.py

```diff
@@ -13,16 +15,20 @@
 
 @command()
 @pass_config
-@argument("platform", required=False, type=str)
-def select(config: Configuration, platform: Optional[str] = None):
+@argument("platform", required=False, type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True))
+def select(config: Configuration, platform: Optional[pathlib.Path] = None):
     """Change current selected platform
 
     if not argument is passed will run in interactive mode"""
+    if config.overridden:
```

Comment:
To remove if ok with comment in main.py
should remove enough complexity to be able to save